src/clue.js

```diff
@@ -1,14 +1,142 @@
 // ITERATION 1
 
+
 // Suspects Collection
-const suspectsArray = [];
+
+const suspectsArray = [ 
+{
+nickname: "mrGreen",
+firstName: "Jacobç",
+lastName: "Green",
+occupation: "Entrepreneur",
+age: "45", 
+description: "He has a lot of connections",
+image: "https://pbs.twimg.com/profile_images/506787499331428352/65jTv2uC.jpeg", 
+color: "green"
+},
+
+{
+nickname: "drOrchid",
+firstName: "Doctor",
+lastName: "Orchid",
+occupation: "Scientist",
+age: "26",
+description: "PhD in plant toxicology. Adopted daughter of Mr. Boddy"
```

Comment:
There needs to be a comma here. Otherwise, the whole file won't be read properly by Jasmine

```suggestion
description: "PhD in plant toxicology. Adopted daughter of Mr. Boddy",
```